app/code/Magento/Catalog/Model/Product/Option/Type/File.php

```diff
@@ -492,7 +489,15 @@ public function copyQuoteToOrder()
      * @param string $url
      * @return $this
      */
-    public function setCustomOptionDownloadUrl($url)
+    public function setCustomOp
```

Comment:
this code looks broken. @evktalo could you please review this code and fix it, so that i can proceed with acceptance of this PR. Thank you